cpp/tests/c_api/bfs_test.c

```diff
@@ -54,15 +55,17 @@ int generic_bfs_test(vertex_t* h_src,
     p_handle, h_src, h_dst, h_wgt, num_edges, store_transposed, &p_graph, &ret_error);
 
   ret_code =
-    cugraph_type_erased_device_array_create(p_handle, INT32, num_seeds, &p_sources, &ret_error);
+    cugraph_type_erased_device_array_create(p_handle, num_seeds, INT32, &p_sources, &ret_error);
```

Comment:
So, we're using (vertex/edge/weight)_tid in the create_graph_test.c, but why are we using INT32 here? Shouldn't we better take the same approach for flexibility & consistency?

Reply:
It would be better to be consistent.  Of course in `create_graph_test.c` we're defining variables but then hard-coding them to the constant INT32.  Better would be to pass the types into the functions in both cases so that the test cases could be parameterized in the main.

These unit tests are relatively new and our standards for the C API testing are not well established.

Reply:
added FIXME in the next commit.